code/modules/reagents/reagents/medicine.dm

```diff
@@ -453,22 +453,26 @@
 	taste_description = "acid"
 	reagent_state = SOLID
 	color = "#004000"
+	metabolism = REM * 1.5
 	overdose = REAGENTS_OVERDOSE
 	scannable = 1 // This is a mostly beneficial chem, it should show up on scanners
 	affects_dead = 1
 
+/datum/reagent/medicine/ryetalyn/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
+    src.on_mob_add(M, alien, effect_multiplier) //I'm going with this to make it both affect dead people for unhusking, and to update on every life tick. Thanks Hydro!
+
 /datum/reagent/medicine/ryetalyn/on_mob_add(mob/living/carbon/M, alien, effect_multiplier) //on_mob_add allows it to act regardless if the human is dead or alive.
-	var/needs_update = M.mutations.len > 0
+    var/needs_update = M.mutations.len > 0
 
-	M.mutations = list()
-	M.disabilities = 0
-	M.sdisabilities = 0
+    M.mutations = list()
+    M.disabilities = 0
+    M.sdisabilities = 0
 
-	// Might need to update appearance for hulk etc.
-	if(needs_update && ishuman(M))
-		var/mob/living/carbon/human/H = M
-		H.update_mutations()
-		H.update_body() //Don't let husks stay wrinkly all the time, we gotta fix them!
+    // Might need to update appearance for hulk etc.
+    if(needs_update && ishuman(M))
```

Comment:
I DID NOT EVEN TOUCH THIS WHY IS THIS SPACES INSTEAD OF TABULATIONS, I ONLY ADDED LINE 462